cdisc_rules_engine/services/data_readers/xpt_reader.py

```diff
@@ -13,10 +15,26 @@ def read(self, data):
         df = self._format_floats(df)
         return df
 
+    def _read_pandas(self, file_path):
+        data = pd.read_sas(file_path, format="xport", encoding="utf-8")
+        return PandasDataset(self._format_floats(data))
+
+    def to_parquet(self, file_path: str) -> str:
+        temp_file = tempfile.NamedTemporaryFile(delete=False, suffix=".parquet")
+        dataset = pd.read_sas(file_path, chunksize=20000, encoding="utf-8")
```

Comment:
In datasetxpt_metadata_reader.py, we switched from xport.v56 to pyreadstat, but here we are using pd.read_sas. What is the difference between these 3 and why are we using different ones in different places?

Reply:
I switched from xport to pyreadstat because it conflicted with the required pandas version for using fastparquet. Both pyreadstat and xport.v56 are used for reading dataset metadata from xpt files, while pandas is used everywhere we actually need to read a file into a pandas dataset